hazelcast-hibernate53/src/test/java/com/hazelcast/hibernate/entity/AnnotatedEntity.java

```diff
@@ -33,14 +30,16 @@ public void setTitle(String title) {
         this.title = title;
     }
 
-    @Id
-    @GeneratedValue(generator="increment")
-    @GenericGenerator(name="increment", strategy = "increment")
+    @javax.persistence.Id
+    @jakarta.persistence.Id
+    @javax.persistence.GeneratedValue(generator = "increment")
+    @jakarta.persistence.GeneratedValue(generator = "increment")
```

Comment:
This is the way... to avoid duplication of the code for tests against Hibernate 5 and 6